tests/config_test.py

```
@@ -0,0 +1,194 @@
+                del sys.modules["tomllib"]
+            if "commit_check.config" in sys.modules:
+                del sys.modules["commit_check.config"]
+
+            # Mock tomllib module to not exist
+            with patch.dict("sys.modules", {"tomllib": None}):
+                # Force import error by patching __import__ for tomllib specifically
+                original_import = __builtins__["__import__"]
+
+                def mock_import(name, *args, **kwargs):
+                    if name == "tomllib":
+                        raise ImportError("No module named 'tomllib'")
```

Ruff (0.13.1)

Avoid specifying long messages outside the exception class

(TRY003)